nessie/repository_test.go

```diff
@@ -0,0 +1,34 @@
+package nessie
+
+import (
+	"context"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+	"github.com/treeverse/lakefs/api/gen/client/repositories"
+)
+
+func TestRepositoryBasicOps(t *testing.T) {
+	ctx := context.Background()
+	const numOfRepos = 5
+
+	// create repositories
+	repos := make([]string, numOfRepos)
+	for i := range repos {
+		repos[i] = createRepositoryUnique(ctx, t)
+	}
+
+	// list repositories - make sure we have the ones we created
+	listedRepos := listRepositories(t, ctx)
```

Comment:
Although this is just a list operation that doesn't change anything, it does return repositories that you don't expect to see here - Repos created by other tests in this run. It does breaks some of the isolation in the tests, but it will still work.

Reply:
true, each isolated test is based on repository - this is with the repositoes test I check for specific names.
I can pass prefix to isolate specific listing, wdyt?